### PR TITLE
[16.0] [MIG] event_contact

### PR DESCRIPTION
**Pull request by edlopen**
Standard migration to 16.0

MT-1840

@yajo @EmilioPascual @Shide @rafaelbn review this PR when you can.

### CODE REVIEW

**Review by yajo — APPROVED** [review]

**Review by EmilioPascual — APPROVED** [review]
@edlopen good job migrating this module.
Funciontal and code review.

**Review by Shide — APPROVED** [review]

**Comment by EmilioPascual** [comment]
@edlopen can you make rebase for run tests again, thank you

File: event_contact/models/event_event.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        comodel_name="res.partner",
+        help="Partners available to attend attendees requests for this event.",
+        compute="_compute_contact_ids",
+        store=True,
+        readonly=False,
+    )
+
+    @api.depends("event_type_id")
+    def _compute_contact_ids(self):
+        self.contact_ids |= self.event_type_id.contact_ids
```
Comment (yajo) [inline event_contact/models/event_event.py]: issue: This seems a bug to me.

```suggestion
        for one in self:
            one.contact_ids |= one.event_type_id.contact_ids
```

**Review by yajo — CHANGES_REQUESTED** [review]

File: event_contact/models/event_event.py (hunk 1 above)
Comment (edlopen) [reply event_contact/models/event_event.py]: Indeed!

**Review by yajo — APPROVED** [review]

**Comment by edlopen** [comment]
Can you check it functionally @fcvalgar? Thanks!

**Review by fcvalgar — APPROVED** [review]
Functional review,

LGTM, thank you @edlopen 

image

image

image

**Pull request merged**